graphdatascience/graph/graph_proc_runner.py

```diff
@@ -44,6 +44,53 @@ def _path(package: str, resource: str) -> ContextManager[pathlib.Path]:
 
             return path(package, resource)
 
+    @client_only_endpoint("gds.graph")
+    @compatible_with("construct", min_inclusive=ServerVersion(2, 1, 0))
+    def construct(
```

Comment:
I see we have duplicated code (should be the same as for the alpha one). Could we perhaps put it in a shared internal method that they both call to avoid this?

Reply:
ACtually, maybe it's not worth it since they're in different classes and so on